File: 1/multiples.jl
function multiples(x,y,z)
    if x % y == 0 || x % z == 0
        return true
    end
end

function sum(x)
    sum = 0
    for i = 1:x-1
        if multiples(i,3,5) == true
            sum = sum + i
        end
    end
    return sum
end

println(sum(1000))

File: 1 - Multiples of 3 and 5/multiples.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

void main(int argc, char *argv[]) {

    int sum = 0;
    for (int i = 1; i < atoi(argv[1]); ++i) {
        if (i % 3 == 0 || i % 5 == 0) {
            sum += i;
        }
    }
    printf("sum = %d\n", sum);

    return;
}

